src/Domain/Entities/AppUser.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Domain.Common;
+using Domain.Enums;
+using System;
+using System.Collections.Generic;
+using System.Text.Json.Serialization;
+
+namespace Domain.Entities
+{
+    public class AppUser : BaseEntity<Guid>
+    {
+        public long TelegramId { get; set; }
+        public Guid LastCommandId { get; set; }
+        public string UserName { get; set; }
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+        public string TelephoneNumber { get; set; }
+        public string Email { get; set; }
+        public Roles Role { get; set; }
+        public DateTime EmploymentStart { get; set; }
+        public DateTime? EmploymentEnd { get; set; }
+        public Guid? PrefferdWorkPlaceId { get; set; }
+        public bool isDeleted { get; set; }
+
+        /* EF Relation */
+        [JsonIgnore]
```

Comment:
Why do we need [JsonIgnore] here?

Reply:
when we output an entity, if we dont use [IsonIgnore] here, then the object that is added for the relation is also output
![photo_2022-02-17_18-15-09](https://user-images.githubusercontent.com/67834974/154511894-56f1d7e3-458a-4334-9dc4-f3cec79152d3.jpg)
![photo_2022-02-17_18-19-43](https://user-images.githubusercontent.com/67834974/154512604-dab4a0e3-c148-4e93-a979-2e3f743d0db6.jpg)
but we can solve it using dto, so i can remove it

Reply:
We have to use DTOs